backend/capellacollab/projects/volumes/routes.py

```diff
@@ -0,0 +1,124 @@
+# SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+# SPDX-License-Identifier: Apache-2.0
+
+import typing as t
+import uuid
+
+import fastapi
+from sqlalchemy import orm
+
+from capellacollab.core import database
+from capellacollab.permissions import models as permissions_models
+from capellacollab.projects import injectables as projects_injectables
+from capellacollab.projects import models as projects_models
+from capellacollab.projects.permissions import (
+    injectables as projects_permissions_injectables,
+)
+from capellacollab.projects.permissions import (
+    models as projects_permissions_models,
+)
+
+from . import crud, exceptions, injectables, models, workspace
+
+router = fastapi.APIRouter(
+    prefix="/volumes",
+    tags=["Projects - Volumes"],
+)
```

Comment:
I think usually we include the path of the route in the parent router, is there any reason for doing it here instead?

Reply:
I thought it might make more sense, but there are pros and cons. I'll put it on the parent router again to keep it consistent.